atomicapp/providers/openshift.py

```diff
@@ -40,6 +40,9 @@
 import logging
 logger = logging.getLogger(__name__)
 
+# If running in an openshift POD via `oc new-app`, the ca file is here
+OPENSHIFT_POD_CA_FILE = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
The question is: do we put provider specific contstants into constants.py? I'm on the fence.